app/components/nationality-select.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { restartableTask, timeout } from 'ember-concurrency';
+
+export default class NationalitySelectComponent extends Component {
+  @service store;
+
+  @restartableTask
+  *searchNationalitiesTask(search = '') {
+    yield timeout(500);
+
+    const query = {
+      sort: 'label',
+    };
+
+    if (search.trim() !== '') {
+      query['filter[label]'] = search;
+    }
+
+    return yield this.store.query('nationality', query);
+  }
```

Comment:
I converted it to a searchable select but I couldn't really try it since I have no data yet :grimacing: